src/Functions/FunctionsConversion_reg.cpp

```diff
@@ -3887,7 +3887,7 @@ SELECT toDecimal256OrNull('42.7', 2), toDecimal256OrNull('invalid', 2)
     /// toUUIDOrNull documentation
     FunctionDocumentation::Description description_toUUIDOrNull = R"(
 Converts an input value to a value of type `UUID` but returns `NULL` in case of an error.
-Like [`toUUID`](#touuid) but returns `NULL` instead of throwing an exception on conversion errors.
+Like [`toUUID`](/sql-reference/functions/uuid-functions#toUUID) but returns `NULL` instead of throwing an exception on conversion errors.
```

Comment:
```suggestion
Like [`toUUID`](/sql-reference/functions/uuid-functions#touuid) but returns `NULL` instead of throwing an exception on conversion errors.
```